docs/LoaderSettingsFile.md

```diff
@@ -0,0 +1,114 @@
+<!-- markdownlint-disable MD041 -->
+[![Khronos Vulkan][1]][2]
+
+[1]: https://vulkan.lunarg.com/img/Vulkan_100px_Dec16.png "https://www.khronos.org/vulkan/"
+[2]: https://www.khronos.org/vulkan/
+
+# Loader Settings File <!-- omit from toc -->
+
+[![Creative Commons][3]][4]
+
+<!-- Copyright &copy; 2025 LunarG, Inc. -->
+
+[3]: https://i.creativecommons.org/l/by-nd/4.0/88x31.png "Creative Commons License"
+[4]: https://creativecommons.org/licenses/by-nd/4.0/
+
+
+## Table of Contents <!-- omit from toc -->
+
+- [Purpose of the Settings File](#purpose-of-the-settings-file)
+- [Settings File Discovery](#settings-file-discovery)
+  - [Windows](#windows)
+  - [Linux/MacOS/BSD/QNX/Fuchsia/GNU](#linuxmacosbsdqnxfuchsiagnu)
+  - [Other Platforms](#other-platforms)
+  - [Exception for Elevated Privileges](#exception-for-elevated-privileges)
+- [Per-Application Settings File](#per-application-settings-file)
+- [File Format](#file-format)
+- [Example Settings File](#example-settings-file)
+  - [Fields](#fields)
+- [Behavior](#behavior)
+
+
+## Purpose of the Settings File
+
+The purpose of the Loader Settings File is to give developers superb control over the
+behavior of the Vulkan-Loader.
+It enables enhanced controls over which layers to load, the order layers in the call chain,
+logging, and which drivers are available.
+
+The Loader Settings File is intended to be used by "Developer Control Panels" for the Vulkan API, such as the Vulkan Configurator, as a replacement for setting debug envrionment variables.
+
+## Settings File Discovery
+
+The Loader Settings File is located by searching in specific file system paths or through
+platform specific mechanisms such as the Windows Registry.
+
+### Windows
+
+The Vulkan Loader first searches the Registry Key  HKEY_CURRENT_USER\SOFTWARE\Khronos\Vulkan\LoaderSettings for a DWORD value whose name is
+a valid path to a file named 'vk_loader_settings.json'.
+If there are no matching values or the file doesn't exist, the Vulkan Loader performs the
+same behavior as described above for the Registry Key  HKEY_LOCAL_MACHINE\SOFTWARE\Khronos\Vulkan\LoaderSettings.
+
+### Linux/MacOS/BSD/QNX/Fuchsia/GNU
+
+The Loader Settings File is located by searching for a file named vk_loader_settings.json in the following locations:
+
+`$HOME/.local/share/vulkan/loader_settings.d/`
```

Comment:
@charles-lunarg  Not sure if that matters, but this location is the only one I am taking into account with Vulkan Configurator, it's where I am always writing vk_loader_settings.json.